app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,62 @@
+class NotificationsController < ApplicationController
+  before_action :set_notification, only: [:show, :edit, :update, :destroy]
+
+  # GET /notifications
+  def index
+    @notifications = Notification.all
+    @page_title = "All Notifications"
```

Comment:
This is good. Could you please add `@page_title` for the other actions ? Could just set it to `subject`, perhaps in a before action.